lib/product-import.coffee

```diff
@@ -230,6 +232,52 @@ class ProductImport
       else
         false
 
+  _updateProductRepeater: (prodToProcess, existingProduct) ->
+    repeater = new Repeater {attempts: 5}
```

Comment:
I think 5 is a good number for retry. Why bother implementing retry and doing it only 2 times? If product is changed by different sync processes often you might run into edge case that 2 retries is not enough.